File: store/src/main/java/fooddelivery/domain/OrderCanceled.java
package fooddelivery.domain;

import fooddelivery.domain.*;
import fooddelivery.infra.AbstractEvent;
import lombok.*;
import java.util.*;
@Data
@ToString
public class OrderCanceled extends AbstractEvent {

    private Long id;
    private String foodId;
    private String customerId;
    private String preference;
}

File: front/OrderCanceled.py
from AbstractEvent import AbstractEvent
import json
from datetime import datetime

class OrderCanceled(AbstractEvent):
    id : int
    foodId : str
    customerId : str
    preference : str
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.foodId = None
        self.customerId = None
        self.preference = None

